### ISSUE TITLE
Crash if applying font to empty string

### ISSUE THREAD

**Issue by ziryanov** [evermeer/AttributedTextView#6]
``` "".font(.systemFont(ofSize: 15)) ``` cause crash

**Comment by ziryanov** [comment]
It seems like this is apple's bug.
```
let string = NSMutableAttributedString(string: "")
let font = string.attribute(NSFontAttributeName, at: 0, effectiveRange: nil) as? UIFont
```
There is crash on second line(only if string is empty and mutable!)

**Comment by ziryanov** [comment]
I've created pull request with fix. Please review and merge.

**Comment by evermeer** [comment]
Thanks for finding and fixing this!
Your PR has been merged and published as version 0.5.1
